updplayer.py

```diff
@@ -157,13 +165,164 @@ def process_schedule(sched_dict):
     return daily_player_dict
 
 
-def search_dictionary(indict, hometeam, awayteam, teamcode, resultdict):
+def build_pitching_stats(keylist, indict):
+    """
+    :param keylist: list of keys in indict dictionary
+    :param indict:  boxscore dictionary from which to extract player data
```

Comment:
I would just extract the keys for the client e.g. using indict.keys() unless there's a reason these might differ?